x-pack/metricbeat/module/meraki/device_health/device_health.go

```diff
@@ -124,6 +102,11 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) error {
 			return fmt.Errorf("getDeviceSwitchports failed; %w", err)
 		}
 
+		err = getDeviceVPNStatuses(m.client, org, devices)
+		if err != nil {
+			return fmt.Errorf("GetVPNStatuses failed; %w", err)
+		}
```

Comment:
Some functions like `getDevicePerformanceScores`, don't always return data. In those cases we just log instead of returning an error. Could this apply for `getDeviceVPNStatuses`?